modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/js/UserNameFields.es.js

```diff
@@ -1,120 +1,141 @@
 	}
 
 	/**
 	 * @inheritDoc
 	 */
 	detached() {
 		super.detached();
-		this.eventHandler_.removeAllListeners();
+
+		this._eventHandler.removeAllListeners();
 	}
 
 	/**
 	 * Updates the user name fields to display the appropriate fields for the
 	 * given locale.
-	 * @param {string} languageId The language id to retrieve user name fields
-	 * for.
+	 *
+	 * @param {string} languageId The language id used when retrieving the user
+	 * name fields.
 	 */
 	updateUserNameFields(languageId) {
 		this._setUp();
 
 		this._getURL(languageId)
 			.then(fetch)
-			.then((response) => response.text())
-			.then(this._insertUserNameFields.bind(this))
-			.then(this._cleanUp.bind(this))
-			.catch(this._handleError.bind(this));
+			.then(
+				(response) => response.text()
+			)
+			.then(
+				this._insertUserNameFields.bind(this)
+			)
+			.then(
+				this._cleanUp.bind(this)
+			)
+			.catch(
+				this._handleError.bind(this)
+			);
 	}
 
 	/**
 	 * Caches the values and maxLength attribute values from the current
-	 * username fields.
-	 * @private
+	 * user name fields.
+	 *
+	 * @protected
 	 */
 	_cacheData() {
 		for (const [name, value] of new FormData(this.formNode)) {
 			const field = this.userNameFieldsNode.querySelector('#' + name);
 
 			if (field) {
-				this.formDataCache_[name] = value;
+				this._formDataCache[name] = value;
 
 				if (field.hasAttribute('maxLength')) {
-					this.maxLengthsCache_[name] = field.getAttribute('maxLength');
+					this._maxLengthsCache[name] = field.getAttribute('maxLength');
 				}
 			}
 		}
 	}
 
 	/**
-	 * Inserts a loading indicator into the dom before the user name fields,
-	 * then hides the user name fields.
-	 * @private
+	 * Inserts a loading indicator before the user name fields and hide
+	 * the user name fields.
+	 *
+	 * @protected
 	 */
 	_createLoadingIndicator() {
 		this.userNameFieldsNode.insertAdjacentHTML(
 			'beforebegin',
-			this.loadingAnimationMarkupText_
+			this._loadingAnimationMarkupText
 		);
 
 		dom.addClasses(this.userNameFieldsNode, 'hide');
 	}
 
 	_cleanUp() {
 		this._removeLoadingIndicator();
+
 		this._populateData();
 	}
 
 	/**
-	 * Returns a promise containing the URL to be used to retrieve the user name
-	 * fields.
+	 * Returns a promise containing the URL to be used to retrieve the user
+	 * name fields.
+	 *
 	 * @param {string} languageId The language id to be set on the URL.
+	 * @protected
 	 * @return {Promise} A promise to be resolved with the constructed URL
-	 * @async
-	 * @private
 	 */
 	_getURL(languageId) {
-		return new Promise((resolve) => {
-			AUI().use(
-				'liferay-portlet-url',
-				(A) => {
-					const url = Liferay.PortletURL.createURL(this.baseURL);
-
-					url.setParameter('languageId', languageId);
-
-					resolve(url);
-				}
-			);
-		});
+		return new Promise(
+			(resolve) => {
+				AUI().use(
+					'liferay-portlet-url',
+					A => {
+						const url = Liferay.PortletURL.createURL(this.baseURL);
```

Comment:
Should we remove the parens from the `resolve` argument above also?

Reply:
Whoops, missed that.  Yes, go ahead and remove that too.  Thanks!